src/helpers/updateHierarchy.helper.js

```diff
@@ -146,6 +147,15 @@ class HierarchyService {
     };
   }
 
+  openForContribution(data, openForContribution) {
+    let instance = this;
```

Comment:
Please do not have the same name for method and parameter

Reply:
fixed @venkateshwarans